release.nix

```diff
@@ -13,8 +13,8 @@ let
   jobs = rec {
 
 
-    tarball =
-      with pkgs;
+    tarball = pkgs.lib.genAttrs systems (system:
+      with import <nixpkgs> { inherit system; };
```

Comment:
We can't have more `import <nixpkgs>`. It's already ignoring the nixpkgs argument passed in to the top-level function. Need to be using that more, not more <nixpkgs>.